frontend/src/locales/en.yml

```diff
@@ -1371,3 +1372,47 @@ subscription:
       title: Enterprise Plan
       label: Early access
       desc: Large organization, has dedicated DBA group to manage database fleet
+schema-review:
```

Comment:
This section should also reflect the change from "schema review" to "schema review policy"